jbcli/jbcli/utils/dockerutil.py

```diff
@@ -71,45 +71,44 @@ def _intersperse(el, l):
     return [y for x in zip([el]*len(l), l) for y in x]
 
 
-def docker_compose(arg, env=None):
+def docker_compose(args, env=None):
     file_args = _intersperse('-f', glob('docker-compose-*.yml'))
     cmd = (
-        ['docker-compose', '-f', 'docker-compose.yml'] + file_args + [arg]
+        ['docker-compose', '-f', 'docker-compose.yml'] + file_args + args
     )
     return check_call(cmd, env=env)
 
 
 def up(env=None):
     """Starts and optionally creates a Docker environment based on
     docker-compose.yml """
-    docker_compose('up', env=env)
+    docker_compose(['up'], env=env)
+
+
+def run_jb(cmd, env=None):
+    docker_compose(['run', 'juicebox'] + cmd, env=env)
 
 
 def destroy():
     """Removes all containers and networks defined in docker-compose.yml"""
-    docker_compose('down')
+    docker_compose(['down'])
 
 
 def halt():
     """Halts all containers defined in docker-compose file."""
-    docker_compose('stop')
+    docker_compose(['stop'])
 
 
 def is_running():
     """Checks whether or not a Juicebox container is currently running.
 
     :rtype: ``bool``
     """
-    running = False
     click.echo('Checking to see if Juicebox is running...')
-    containers = client.containers.list(all=True)
-    if containers:
-        for container in containers:
-
-            if 'juicebox' in container.name and get_state(
-                    container.name) == 'running':
-                running = True
-    return running
+    containers = client.containers.list()
```

Comment:
this code was needlessly saying "list containers, even ones that aren't running" and then explicitly checking if they were running. Instead, I changed this code to use the default `list` behavior, which is to not include stopped containers.